DataFormats/Detectors/TRD/include/DataFormatsTRD/NoiseCalibration.h

```diff
@@ -27,6 +27,53 @@ namespace trd
 /// \brief Simple noise status bit for each MCM of the TRD
 /// \author Ole Schmidt
 
+class ChannelInfo
+{
+ public:
+  enum Flag : uint8_t {
+    IsSet,
+    IsOK,
+    IsMasked,
+    IsNoise,
+    LowMean,
+    HighMean,
+    LowRMS,
+    HighRMS
+  };
+
+  ChannelInfo() = default;
+
+  bool isDummy() const { return !(mFlags & (1 << IsSet)); }
+  bool getFlag(uint8_t i) const { return (mFlags & (1 << i)); }
+  float getMean() const { return mMean; }
+  float getRMS() const { return mRMS; }
+  int getFlags() const { return mFlags; }
+
+  void setMean(float mean) { mMean = mean; }
+  void setRMS(float rms) { mRMS = rms; }
+  void setFlags(uint8_t flags) { mFlags = flags; }
+
+ private:
+  float mMean{0.f};
+  float mRMS{0.f};
+  uint8_t mFlags{0};
```

Comment:
When I thought about this struct, I considered to include the number of seen digits for each pad. This adds to the data volume, but might indicate data transfer problems. But it might not be worth it.

Reply:
I use one of the flags just to indicate whether at least one digit was seen from one channel. But we could also add a counter there